launch/launch/actions/execute_local.py

```diff
@@ -98,7 +100,7 @@ def __init__(
             SomeActionsType,
             Callable[[ProcessExited, LaunchContext], Optional[SomeActionsType]]
         ]] = None,
-        respawn: bool = False,
+        respawn: Union[bool, SomeSubstitutionsType] = False,
```

Comment:
@kosuke55 nit: update the docstring to reflect that it now supports substitutions.

Reply:
@hidmic Updated docstring https://github.com/ros2/launch/pull/569/commits/16198d710b2a11efaf8d8ec21417df2f48965c0c